src/scripts/index.js

```diff
@@ -0,0 +1,20 @@
+import createTag from "./utils/createTag.js";
+
+//Send request DOM API elements
+const divSend = createTag({ className: "div-send-request" });
+const spanSelectAndInput = createTag({
+  tagName: "span",
+  className: "span-select-and-input",
```

Comment:
We prefer to use semantic tags in HTML, I have no idea why we need this tag. From CSS perspective is not required

Reply:
This was my thought, that somehow we will need it for CSS styling. Span deleted.